test/runtime/kafka.go

```diff
@@ -139,4 +139,38 @@ var _ = Describe("RuntimeValidatedKafka", func() {
 		res = vm.Exec(consumer(disallowTopic, MaxMessages))
 		res.ExpectFail("Kafka consumer can access to disallowTopic")
 	})
+
+	It("Kafka Policy Role Ingress", func() {
+		_, err := vm.PolicyImportAndWait(vm.GetFullPath("Policies-kafka-Role.json"), 300)
+		Expect(err).Should(BeNil(), "Expected nil got %s while importing policy Policies-kafka-Role.json", err)
+
+		endPoints, err := vm.PolicyEndpointsSummary()
+		Expect(err).Should(BeNil(), "Expect nil. Failed to apply policy on all endpoints with error :%s", err)
+		Expect(endPoints[helpers.Enabled]).To(Equal(1), "Expected 1 endpoint to be policy enabled. Policy enforcement failed")
+		Expect(endPoints[helpers.Disabled]).To(Equal(2), "Expected 2 endpoint to be policy disabled. Policy enforcement failed")
+
+		createTopic(allowedTopic)
+		createTopic(disallowTopic)
+
+		res := vm.ContainerExec("client",
+			"/opt/kafka/bin/kafka-topics.sh --list --zookeeper zook:2181")
+		res.ExpectSuccess("Cannot get kafka topics")
+
+		By("By sending produce/consume request on topic `allowedTopic`")
+		ctx, cancel := context.WithCancel(context.Background())
+		defer cancel()
+		data := vm.ExecContext(ctx, consumer(allowedTopic, MaxMessages))
+
+		helpers.Sleep(5)
```

Comment:
Remove sleep?

Reply:
#3116 I opened this to add correctly waiting for broker to be up.Just waiting for endpoints won't work. I am adding a comment here for now and keeping the Sleep, same as what is done for Runtime test `Kafka Policy Ingress`